SAMtags.tex

```diff
@@ -75,32 +76,36 @@ \section{Standard tags}
   {\tt H1} & i & Number of 1-difference hits (see also {\tt NM}) \\
   {\tt H2} & i & Number of 2-difference hits \\
   {\tt HI} & i & Query hit index \\
-  {\tt IH} & i & Number of stored alignments in SAM that contains the query in the current record\\
+  {\tt IH} & i & Number of stored alignments in SAM that contains the query in the current record \\
   {\tt LB} & Z & Library \\
-  {\tt MC} & Z & CIGAR string for mate/next segment\\
+  {\tt MC} & Z & CIGAR string for mate/next segment \\
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
+  {\tt MI} & Z & Molecular-barcode sequence or identifier \\
   {\tt MQ} & i & Mapping quality of the mate/next segment \\
-  {\tt NH} & i & Number of reported alignments that contains the query in the current record\\
+  {\tt NH} & i & Number of reported alignments that contains the query in the current record \\
   {\tt NM} & i & Edit distance to the reference \\
   {\tt OC} & Z & Original CIGAR \\
   {\tt OP} & i & Original mapping position \\
   {\tt OQ} & Z & Original base quality \\
+  {\tt OX} & Z & Raw (uncorrected) unique molecular barcode bases \\
   {\tt PG} & Z & Program \\
   {\tt PQ} & i & Phred likelihood of the template \\
   {\tt PT} & Z & Read annotations for parts of the padded read sequence \\
   {\tt PU} & Z & Platform unit \\
-  {\tt QT} & Z & Barcode ({\tt BC} or {\tt RT}) phred-scaled base qualities \\
   {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag \\
+  {\tt QT} & Z & Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag \\
+  {\tt QX} & Z & Quality score of the unique molecular identifier in the {\tt RX} tag \\
   {\tt R2} & Z & Sequence of the mate/next segment in the template \\
   {\tt RG} & Z & Read group \\
   {\tt RT} & Z & Barcode sequence (deprecated; use {\tt BC} instead) \\
+  {\tt RX} & Z & Sequence bases of the unique molecular identifier \\
```

Comment:
Should this note `, possibly corrected` or something like that to capture the discussion in this thread?